ui/desktop/app/routes/scopes/scope/projects/sessions/index.js

```diff
@@ -105,7 +110,7 @@ export default class ScopesScopeProjectsSessionsIndexRoute extends Route {
       this.allSessions = await this.store.query(
```

Comment:
Just wanted to note we didn't include the parent scope filter for either of the sessions call

Reply:
Sorry, that was actually part of my changes yesterday and I did not include them. Adding them back now.